mysqld_exporter.go

```diff
@@ -46,6 +46,14 @@ var (
 		"collect.perf_schema.indexiowaitstime", false,
 		"Collect time metrics from performance_schema.table_io_waits_summary_by_index_usage",
 	)
+	perfEventsStatements = flag.Bool(
+		"collect.perf_schema.eventsstatements", false,
```

Comment:
Is this a one-to-one mapping to an existing MySQL name? Otherwise `eventsstatements` sounds like a double plural.